### ISSUE TITLE
Configs for umbrella apps not part of releases

### ISSUE THREAD

**Issue by jeffweiss** [bitwalker/conform#52]
When using an umbrella application, the config files for any of the apps (but especially the primary application) do not get packaged as part of the exrm release.

I was able to replicate this with https://github.com/bitwalker/exrm-umbrella-test.
```shell
$ MIX_ENV=prod mix conform.new
$ MIX_ENV=prod mix conform.configure
$ cd apps/master_app
$ MIX_ENV=prod mix release
$ find . -type f -name "*.conf"
```

I expected there to be a `master_app.conf` in `rel/master_app/releases/0.0.1/`

**Comment by jeffweiss** [comment]
Ah, I figured out that if I explicitly recompile `conform` after `exrm` is compiled, the release plugin will be registered and the .conf file will be copied to the release as expected.

```shell
 jeff  ⋯  exrm-umbrella-test  apps  master_app  ls -l rel/master_app/releases/0.0.1                                                                                                              master
total 40704
-rwx------+ 1 jeff  staff      7289 Nov 20 15:39 master_app.bat
-rw-r--r--+ 1 jeff  staff     26899 Nov 20 15:39 master_app.boot
-rw-r--r--+ 1 jeff  staff       479 Nov 20 15:39 master_app.rel
-rw-r--r--+ 1 jeff  staff     35584 Nov 20 15:39 master_app.script
-rwx------+ 1 jeff  staff     19623 Nov 20 15:39 master_app.sh
-rw-r--r--+ 1 jeff  staff  20693319 Nov 20 15:39 master_app.tar.gz
-rw-r--r--+ 1 jeff  staff     26899 Nov 20 15:39 start.boot
-rw-r--r--+ 1 jeff  staff      5312 Nov 20 15:39 start_clean.boot
-rw-r--r--+ 1 jeff  staff        80 Nov 20 15:39 sys.config
-rw-r--r--+ 1 jeff  staff       433 Nov 20 15:39 vm.args
 jeff  ⋯  exrm-umbrella-test  apps  master_app  MIX_ENV=prod mix release.plugins                                                                                                                 master
protocol.consolidation         # Performs protocol consolidation for your release.
appup                          # Generates a .appup for each dependency in your project
 jeff  ⋯  exrm-umbrella-test  apps  master_app  MIX_ENV=prod mix deps.compile conform                                                                                                            master
==> conform
Compiled lib/mix/plugins/exrm_conform.ex
Generated conform app
 jeff  ⋯  exrm-umbrella-test  apps  master_app  MIX_ENV=prod mix release.plugins                                                                                                                 master
protocol.consolidation         # Performs protocol consolidation for your release.
appup                          # Generates a .appup for each dependency in your project
conform                        # Generates a .conf for your release
 jeff  ⋯  exrm-umbrella-test  apps  master_app  MIX_ENV=prod mix release                                                                                                                         master
Building release with MIX_ENV=prod.

You have dependencies (direct/transitive) which are not in :applications!
The following apps should to be added to :applications in mix.exs:

        conform -> neotoma => neotoma is missing from conform

Continue anyway? Your release may not work as expected if these dependencies are required! [Yn]:

Compiling src/conf_parse.peg
PEG manifest updated
==> conform
Compiled src/conf_parse.erl
lib/conform/schema.ex:1: warning: redefining module Conform.Schema
Compiled lib/conform/schema/transform.ex
Compiled lib/conform/schema/validator.ex
Compiled lib/conform/sysconfig.ex
Compiled lib/conform/schema/mapping.ex
Compiled lib/conform/type.ex
Compiled lib/conform.ex
Compiled lib/conform/types/enum.ex
Compiled lib/conform/conf.ex
Compiled lib/conform/validators/range_validator.ex
lib/mix/plugins/exrm_conform.ex:2: warning: redefining module ReleaseManager.Plugin.Conform
lib/mix/tasks/conform.configure.ex:1: warning: redefining module Mix.Tasks.Conform.Configure
lib/mix/tasks/conform.archive.ex:1: warning: redefining module Mix.Tasks.Conform.Archive
Compiled lib/conform/parse.ex
lib/mix/tasks/conform.new.ex:1: warning: redefining module Mix.Tasks.Conform.New
lib/mix/tasks/conform.release.ex:1: warning: redefining module Mix.Tasks.Conform.Release
lib/mix/tasks/conform.effective.ex:1: warning: redefining module Mix.Tasks.Conform.Effective
Compiled lib/mix/plugins/exrm_conform.ex
Compiled lib/mix/tasks/conform.archive.ex
Compiled lib/mix/tasks/conform.release.ex
Compiled lib/conform/utils/utils.ex
Compiled lib/mix/tasks/conform.configure.ex
Compiled lib/mix/tasks/conform.new.ex
Compiled lib/conform/utils/code.ex
Compiled lib/mix/tasks/conform.effective.ex
Compiled lib/conform/schema.ex
Compiled lib/conform/translate.ex
Generated conform app
Generated escript conform with MIX_ENV=prod
==> The release for master_app-0.0.1 is ready!
==> You can boot a console running your release with `$ rel/master_app/bin/master_app console`
 jeff  ⋯  exrm-umbrella-test  apps  master_app  ls -l rel/master_app/releases/0.0.1                                                                                                              master
total 54296
-rwxr-xr-x+ 1 jeff  staff   2323886 Nov 20 15:40 conform
-rwx------+ 1 jeff  staff      7289 Nov 20 15:40 master_app.bat
-rw-r--r--+ 1 jeff  staff     26977 Nov 20 15:40 master_app.boot
-rw-r--r--+ 1 jeff  staff       136 Nov 20 14:54 master_app.conf
-rw-r--r--+ 1 jeff  staff       479 Nov 20 15:40 master_app.rel
-rw-r--r--+ 1 jeff  staff      2763 Nov 20 14:53 master_app.schema.exs
-rw-r--r--+ 1 jeff  staff     35698 Nov 20 15:40 master_app.script
-rwx------+ 1 jeff  staff     19623 Nov 20 15:40 master_app.sh
-rw-r--r--+ 1 jeff  staff  25319728 Nov 20 15:40 master_app.tar.gz
-rw-r--r--+ 1 jeff  staff     26977 Nov 20 15:40 start.boot
-rw-r--r--+ 1 jeff  staff      5312 Nov 20 15:40 start_clean.boot
-rw-r--r--+ 1 jeff  staff        80 Nov 20 15:40 sys.config
-rw-r--r--+ 1 jeff  staff       433 Nov 20 15:40 vm.args
```

**Comment by bitwalker** [comment]
Ah that makes sense, I'll have to make a change in how exrm detects plugins, but it's fairly straightforward, should have a fix up this weekend.

**Comment by bitwalker** [comment]
This is now fixed in conform 1.0.0-rc3
